src/utils/get-input-from-env.ts

```diff
@@ -26,6 +26,9 @@ export const getInputFromEnv: GetInputFromEnvFn = ({
       `Expected ${type} for input ${name}, but got ${typeof value}`
     );
   }
+  if (type === "string" && !required) {
+    return value || null;
+  }
```

Comment:
Note that this fixes a bug with the other optional strings. Because we do this:

```
  env:
    API_TOKEN: ${{ inputs.api-token }}
    GITHUB_TOKEN: ${{ inputs.github-token }}
    APP_URL: ${{ inputs.app-url }}
    TESTS_FILE: ${{ inputs.tests-file }}
    MAX_RETRIES_ON_FAILURE: ${{ inputs.max-retries-on-failure }}
    PARALLEL_TASKS: ${{ inputs.parallel-tasks }}
    LOCALHOST_ALIASES: ${{ inputs.localhost-aliases }}
    MAX_ALLOWED_COLOR_DIFFERENCE: ${{ inputs.max-allowed-color-difference }}
    MAX_ALLOWED_PROPORTION_OF_CHANGED_PIXELS: ${{ inputs.max-allowed-proportion-of-changed-pixels }}
    USE_DEPLOYMENT_URL: ${{ inputs.use-deployment-url }}
    ENVIRONMENT_TO_TEST: ${{ inputs.environment-to-test }}
    TEST_SUITE_ID: ${{ inputs.test-suite-id }}
    METICULOUS_TELEMETRY_SAMPLE_RATE: "0.01"
```

Any inputs the user doesn't specify come through as "". Previously we were keeping them as "" rather than converting to null.